sphinx/source/docs/dev_guide/building.rst

```diff
@@ -119,12 +119,12 @@ Building BokehJS with SBT
         The ``sbt`` build system is experimental and not integrated with
         ``setup.py``, so it should be used with caution.
 
-As an alternative to Grunt, you can use `SBT`_ to build BokehJS. To start,
+As an alternative to Gulp, you can use `SBT`_ to build BokehJS. To start,
```

Comment:
@mattpap uses it to unify his development process with that for bokeh-scala. If he wants to support it for his own use, that's fine, but we should not mention it in the docs. This entire section ought to be removed.